### PR TITLE
Add publications model and management

### PR DESCRIPTION
**Pull request by simaob**

### CODE REVIEW

File: app/decorators/publication_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class PublicationDecorator < Draper::Decorator
+  delegate_all
+
+  def title_link
```
Comment (tsubik) [inline app/decorators/publication_decorator.rb]: You can add file link here too. Take a look at DataUploadDecorator, on Publications show page there should be downloadable link for the file.

**Comment by tsubik** [comment]
We gonna have to add more validation rules for things like thumbnails, like image minimum size. Also it could be processed by ImageMagick to a valid size after uploading. This is for other story though.

**Comment by simaob** [comment]
Yeah, I'll add a task about validations, and added the suggested file_link.

**Comment by tsubik** [comment]
:+1:

**Pull request merged**